modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3353,11 +3353,7 @@ private void _validateValues(
 				objectDefinitionId, entry.getKey());
 		}
 		catch (NoSuchObjectFieldException noSuchObjectFieldException) {
-			if (_log.isDebugEnabled()) {
-				_log.debug(noSuchObjectFieldException);
-			}
-
-			return;
+			throw noSuchObjectFieldException;
```

Comment:
@TaiPhamLR we can remove the `try/catch` and do: 

```
ObjectField objectField = _objectFieldLocalService.getObjectField(objectDefinitionId, entry.getKey());
```

Reply:
Thanks @carolmariaabb. I've fixed it.